dcpquery/db/__main__.py

```diff
@@ -2,19 +2,28 @@
 This module provides access to administrative commands for the DCP Query Service database.
 """
 
-import os, sys, argparse, logging
+import os, sys, argparse, logging, json
 
 from dcplib.etl import DSSExtractor
 
 from .. import config
 from ..etl import transform_bundle, load_bundle
 from . import DCPQueryDBManager
 
+default_test_query = {
+    "query": {
+        "match": {
+            "files.project_json.project_core.project_short_name": "HPSI human cerebral organoids"
```

Comment:
This will only contain 10x bundles. I'll create a ticket to add smartseq2 bundles to the test data as well.